edx_exams/apps/core/models.py

```diff
@@ -255,6 +255,30 @@ def get_attempt_for_user_with_attempt_number_and_resource_id(cls, user_id, attem
             return None
 
 
+class AssessmentControlResult(TimeStampedModel):
+    """
+    Information about the exam attempt provided by the proctoring provider
+    though the ACS API
+
+    .. no_pii:
+    """
+
+    attempt = models.ForeignKey(ExamAttempt, on_delete=models.CASCADE)
+
+    action_type = models.CharField(max_length=64, choices=[('terminate', 'terminate')])
+
+    incident_time = models.DateTimeField()
+
+    severity = models.DecimalField(max_digits=3, decimal_places=2)
+
+    reason_code = models.CharField(max_length=64)
```

Comment:
Yeah we should probably store them somewhere... In a perfect world that would be configurable by provider outside the codebase along with what codes are error and what codes are success. Sounds like a followup for when we add a second provider to me. Maybe we just add some constants or something to map this right now?

Leaning toward a simple solution right now which could be:

- "1" is a success and anything else is an error
- We keep a map of display labels on the backend for error codes and return that display label instead of the code in the serializer.

Reply:
Makes sense to me, I'll keep this in mind.